libraries/engage/back-in-stock/components/Subscriptions/components/Subscription/index.jsx

```diff
@@ -139,6 +145,13 @@ const Subscription = ({
           </div>
         </div>
         <div className={classNames(styles.baseContainerRow)}>
+          {product?.characteristics?.map(characteristic =>
+            <div
+              key={getCharacteristicString(characteristic)}
```

Comment:
i think the approach from the favorites list is a bit better. with this you can avoid additional function calls

https://github.com/shopgate/pwa/blob/master/libraries/engage/favorites/components/Item/ItemCharacteristics.jsx#L28